backend/main.py

```
@@ -44,14 +57,36 @@ def get_db():
         db.close()
 
 
-class LoginRequest(BaseModel):
-    email: str
-    password: str
+
+
+
+def authenticate_user(db: Session, username: str, password: str):  
```

Ruff (0.8.2)

Missing return type annotation for public function `authenticate_user`

(ANN201)